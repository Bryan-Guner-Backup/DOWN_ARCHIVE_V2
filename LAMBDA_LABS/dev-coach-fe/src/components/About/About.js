import React from 'react';
import Navigation from '../Landing/Navigation/Navigation';
import Search from './Search/Search';
import How from './How/How';
import Testimonials from './Testimonials/Testimonial';
import AboutContainer from './AboutStyles';
import Footer from '../Landing/Footer/Footer';


const About = () => {
  return (
    <div>
      
      <Navigation />
      <AboutContainer className='aboutContainer'>
        <Search />
        <How />
        <Testimonials />
      </AboutContainer>
      <Footer />
    </div>
  );
};

export default About;