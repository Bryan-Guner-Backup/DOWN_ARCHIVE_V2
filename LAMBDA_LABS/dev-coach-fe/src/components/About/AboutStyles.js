import styled from 'styled-components';
import devices from "../../utils/devices"

export default styled.div`

padding: 0 10%;
background-image: url("https://i.ibb.co/7y084jG/green-dots.png");
background-color: #E6F3E6;
background-position: center;
background-repeat: repeat-y;
background-size: cover;


h1{
  color: #4FAD65;
    font-size: 3rem;
    margin: 2rem 0 1rem 0
}
h2{
    margin: 2rem 0 1rem 0
}
.navigation {
   .list-items .list-item a {
   color: black;
 }
}
@media${devices.large}{
  padding: 0 15rem;
}
`;
