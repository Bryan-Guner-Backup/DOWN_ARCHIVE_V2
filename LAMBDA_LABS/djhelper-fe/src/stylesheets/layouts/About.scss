//desktop

@media(min-width: 1024px){

.about-page {
  background: black;
  padding-bottom: 2rem;
  color: $purpleFromImage;
  h1 {
    text-align: center;
    font-weight: bold;
    font-size: 5rem;
    color: white;
    padding: 2rem;
    @media (max-width: 1000px) {
      font-size: 3rem;
    }
  }
  .hero {
    
    background-image:url('../../images/desktop_about_-_header_image.png');
    background-size: cover;
    background-repeat: no-repeat;
    min-height: 60vh;
    width: 100%;
    display: flex;
    justify-content: flex-start;
    align-items: center;
    margin-bottom: 60px;
  }
 
 .aboutTop{
   position: absolute;
   top: 260px;
   padding-bottom: 30px;
   h1{
     padding-left:93px;
     padding-right:0px;
     margin:20px;
   }
 }

  .aboutBox{
    min-height: 30vh;
    width: 160vh;
    display:flex;
    justify-content: center;
    background-color: #525252;
    align-items: center;
    margin-left: 20vh;
    padding: 5vh;
    border-radius: 5px;
      p{
        color:black;
        font-size: 14px; 
    }
  
  }
  .title {
    text-align: left;
    margin: 0rem 12rem 0rem 12rem;
   
  
   
  }
  .djTeam{
    display: flex;
    flex-direction: row;
    margin: 0rem 12rem 0rem 12rem;
    justify-content: left;
    
  
  }
  .IOSTeam, .UXTeam, .DataTeam{
    width: 332px;
    height:432px;
    border: 1px solid black;
    margin: 15px;
    border-radius: 3px;
    background-color: #333333;
   
    img{
      width:210px;
      height:210px;
      border-radius: 150px;
      margin: 30px 0px 3px 60px;
      
    }
    h3{
      color: white;
      font-size: 2.5rem;
     text-align: center;
      margin-top: 2rem;

    }
    p{
      color:white;
      font-size: 16px;
      text-align: center;
    }
  }
  .webteam {
    
    width: 332px;
    height:432px;
    border: 1px solid black;
    margin: 15px;
    border-radius: 3px;
    background-color: #333333;
   
    img{
      width:180px;
      height:180px;
      border-radius: 150px;
      margin: 20px 0px 0px 50px;
    
    }
    h3{
      color: white;
      font-size: 2.5rem;
      text-align: center;
      margin-top: 2rem;
    }
    p{
      color:white;
      font-size: 16px;
      text-align: center;
    }
  }
}
  }

/////////////////////////tablet////////////////
@media (max-width:1024px), (min-width:480){
  .about-page {
    background: black;
    padding-bottom: 2rem;
    color: $purpleFromImage;
    h1 {
      text-align: center;
      font-weight: bold;
      font-size: 5rem;
      color: white;
      padding: 2rem;
      @media (max-width: 1000px) {
        font-size: 3rem;
      }
    }
    .hero {
      
      background-image:url('../../images/desktop_about_-_header_image.png');
      background-size: cover;
      background-repeat: no-repeat;
      min-height: 60vh;
      width: 100%;
      display: flex;
      justify-content: center;
      align-items: center;
      margin:0px;
    
    }


    .aboutBox{
    
      min-height: 30vh;
      width: 100%;
      display:flex;
      justify-content: center;
      background-color:#525252;
      align-items: center;
      margin: 0px;
      padding: 5vh;
      border-radius: 5px;
        p{
          color:black;
          font-size: 14px; 
      }
    
    }
    .title {
      text-align:center;
     
    }
    .djTeam{
      max-width: 100%;
      display: flex;
      flex-direction: row;
      flex-wrap: wrap;
      margin: 0rem 12rem 0rem 12rem;
      justify-content: center;
      
    
    }
    .DataTeam, .IOSTeam, .UXTeam{
      max-width: 100%;
      width: 332px;
      height:432px;
      border: 1px solid black;
      margin: 15px;
      border-radius: 3px;
      background-color: #333333;
     
      img{
        max-width: 100%;
        width:210px;
        height:210px;
        border-radius: 150px;
        margin: 30px 0px 3px 60px;
       
        
      }
      h3{
        color: white;
        font-size: 2.5rem;
       text-align: center;
        margin-top: 2rem;
  
      }
      p{
        color:white;
        font-size: 16px;
        text-align: center;
      }
    }
    .webteam {
      
      width: 332px;
      height:432px;
      border: 1px solid black;
      margin: 15px;
      border-radius: 3px;
      background-color: #333333;
      justify-content: center;
     
      img{
        width:180px;
        height:180px;
        border-radius: 150px;
        margin: 30px 0px 0px 75px;
      }
      h3{
        color: white;
        font-size: 2.5rem;
        text-align: center;
        margin-top: 2rem;
      }
      p{
        color:white;
        font-size: 16px;
        text-align: center;
      }
    }
  }
  

  //end of media query
}
///////////////////// Phone /////////////////////
@media (max-width:480px){
  .about-page {
    background: black;
    padding-bottom: 2rem;
    color: $purpleFromImage;
    h1 {
      text-align: center;
      font-weight: bold;
      font-size: 5rem;
      color: white;
      padding: 2rem;
      @media (max-width: 1000px) {
        font-size: 3rem;
      }
    }
    .hero {
      display:none;
    
    }
    .aboutBox{
    
      min-height: 30vh;
      width: 100%;
      display:flex;
      justify-content: center;
      background-color:#525252;
      align-items: center;
     // margin-left: 20vh;
      padding: 5vh;
      border-radius: 5px;
        p{
          color:black;
          font-size: 14px; 
      }
    
    }
    .title {
      text-align:center;
     
    }
    .djTeam{
      max-width: 100%;
      display: flex;
      flex-direction: row;
      flex-wrap: wrap;
      margin: 0rem;
      justify-content: center;
      
    
    }
    .UXTeam, .IOSTeam, .DataTeam{
      width: 332px;
      height:432px;
      border: 1px solid black;
      margin: 15px;
      border-radius: 3px;
      background-color:#333333;
      justify-content: center;
      align-items:center;
     
      img{
        width:180px;
        height:180px;
        border-radius: 150px;
        margin-left: 70px;
        margin-right:20px;
        
      }
      h3{
        color: white;
        font-size: 2.5rem;
       text-align: center;
        margin-top: 2rem;
  
      }
      p{
        color:white;
        font-size: 16px;
        text-align: center;
      }
    
    }
    
    
    
    
    
    .webteam{
      width: 332px;
      height:432px;
      border: 1px solid black;
      margin: 15px;
      border-radius: 3px;
      background-color: #333333;
      justify-content: center;
      align-items: center;
     
      img{
        width:180px;
        height:180px;
        border-radius: 150px;
        margin-left: 70px;
        margin-right: 40px;
      }
      h3{
        color: white;
        font-size: 2.5rem;
       text-align: center;
        margin-top: 2rem;
  
      }
      p{
        color:white;
        font-size: 16px;
        text-align: center;
      }
    }
   
  }
}