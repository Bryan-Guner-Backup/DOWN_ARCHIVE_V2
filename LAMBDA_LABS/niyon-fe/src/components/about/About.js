import React from 'react'
import Navbar from '../navbar/Navbar'
import PersonCard from './PersonCard'

function About (props) {
  return (
    <div>
      <Navbar />
      <div className="container" data-test="aboutUs-container">
        <h1>Meet the Crew</h1>
        <PersonCard />
      </div>
    </div>
  )
}

export default About
