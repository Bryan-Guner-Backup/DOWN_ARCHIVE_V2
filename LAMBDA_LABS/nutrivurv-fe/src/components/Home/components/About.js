import React from "react";
// import GitHubCalendar from "react-github-calendar";
// import ReactTooltip from "react-tooltip";
import Footer from "./LandingPage/components/Footer";

const About = () => {
  return (
    <div>
      <br />
      <br />
      <div className="d-flex justify-content-center py-4">
        <div>
          <h2>Our Team</h2>
        </div>
      </div>
      <div className="d-flex flex-wrap justify-content-around rounded mx-auto py-4 my-4 about-div">
        <div className="p-2 my-2 mx-2 pb-4 pt-4 mb-4 justify-content-center bg-light rounded about-img shadow">
          <div className="d-block">
            <div className="d-flex justify-content-center mb-4">
              <img
                src="https://github.com/dave-vazquez.png"
                alt="Dave Vazquez"
                className="img-responsive rounded-circle"
                width="150px"
              />
            </div>
            <div className="d-flex flex-column justify-content-center text-center">
              <h4>Dave Vazquez</h4>
              <h6 className="font-weight-bold my-2">Team Lead / Full Stack Web Developer</h6>
            </div>
            <div className="d-flex justify-content-center">
                <i className="fa fa-github fa-large text-dark mt-1 mr-3"></i>
              <a href="https://github.com/dave-vazquez">Github</a>
            </div>
          </div>
        </div>

        <div className=" p-2 my-2 mx-2 pt-4 mb-4 pb-4 justify-content-center bg-light rounded about-img  shadow">
          <div className="d-block">
            <div className="d-flex justify-content-center mb-4">
              <img
                src="https://github.com/alesslongaretti.png"
                alt="Alessandra Longaretti"
                className="img-responsive rounded-circle"
                width="150px"
              />
            </div>
            <div className="d-flex justify-content-center flex-column text-center">
              <h4>Alessandra Longaretti</h4>
              <h6 className="font-weight-bold my-2">Full Stack Web Developer</h6>
            </div>
            <div className="d-flex justify-content-center">
                <i className="fa fa-github fa-large text-dark mt-1 mr-3"></i>
              <a href="https://github.com/alesslongaretti">Github</a>
            </div>
          </div>
        </div>
        <div className="p-2 my-2 mx-2 pt-4 mb-4 pb-4 justify-content-center bg-light rounded about-img shadow">
          <div className="d-block">
            <div className="d-flex justify-content-center mb-4">
              <img
                src="https://github.com/taty2010.png"
                alt="Taty Novell"
                className="img-responsive rounded-circle"
                width="150px"
              />
            </div>
            <div className="d-flex justify-content-center flex-column text-center">
              <h4>Taty Novell</h4>
              <h6 className="font-weight-bold my-2">Full Stack Web Developer</h6>
            </div>
            <div className="d-flex justify-content-center">
                <i className="fa fa-github fa-large text-dark mt-1 mr-3"></i>
              <a href="https://github.com/taty2010">Github</a>
            </div>
          </div>
        </div>

        <div className=" p-2 mt-3 mx-2 pt-4 mb-4 pb-4 justify-content-center bg-light rounded about-img  shadow">
          <div className="block">
            <div className="d-flex justify-content-center mb-4">
              <img
                src="https://github.com/ccsmith13.png"
                alt="Christine Smith"
                className="img-responsive rounded-circle"
                width="150px"
              />
            </div>
            <div className="d-flex justify-content-center flex-column text-center">
              <h4>Christine Smith</h4>
              <h6 className="font-weight-bold my-2">Full Stack Web Developer</h6>
            </div>
            <div className="d-flex justify-content-center">
                <i className="fa fa-github fa-large text-dark mt-1 mr-3"></i>
              <a href="https://github.com/ccsmith13">Github</a>
            </div>
          </div>
        </div>

        <div className="p-2 mt-3 mx-2 pt-4 mb-4 pb-4 justify-content-center bg-light rounded about-img  shadow">
          <div className="d-block">
            <div className="d-flex justify-content-center mb-4">
              <img
                src="https://github.com/afrodo1.png"
                alt="Dominic Bridgette"
                className="img-responsive rounded-circle"
                width="150px"
              />
            </div>
            <div className="d-flex justify-content-center flex-column text-center">
              <h4>Dominic Bridgette</h4>
              <h6 className="font-weight-bold my-2">Full Stack Web Developer</h6>
            </div>
            <div className="d-flex justify-content-center">
                <i className="fa fa-github fa-large text-dark mt-1 mr-3"></i>
              <a href="https://github.com/afrodo1">Github</a>
            </div>
          </div>
        </div>

        <div className="p-2 mt-3 mx-2 pt-4 mb-4 pb-4 justify-content-center bg-light rounded about-img  shadow">
          <div className="d-block">
            <div className="d-flex justify-content-center mb-4">
              <img
                src="https://github.com/dillonp23.png"
                alt="Dillon Petretti"
                className="img-responsive rounded-circle"
                width="150px"
              />
            </div>
            <div className="d-flex justify-content-center flex-column text-center">
              <h4>Dillon Petretti</h4>
              <h6 className="font-weight-bold my-2">iOS Developer</h6>
            </div>
            <div className="d-flex justify-content-center">
                <i className="fa fa-github fa-large text-dark mt-1 mr-3"></i>
              <a href="https://github.com/dillonp23">Github</a>
            </div>
          </div>
        </div>
      </div>
      <Footer />
    </div>
  );
};

export default About;
