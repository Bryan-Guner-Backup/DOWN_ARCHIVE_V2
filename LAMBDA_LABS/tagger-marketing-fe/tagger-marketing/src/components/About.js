//import dependencies
import React from "react";

import { Player } from "video-react";

//set Component Function
export default function About() {
  //render the component
  return (
    <div className="wrapper">
      <div className="about">
        <div className="about-content-container">
          <div className="register-container">
            <div className="tagline">
              <h3>
                Tagger is not just a faster email, <br/>
                it's Smarter Email.
              </h3>
            </div>
            <div className="about-form-container">
              <h3 className="heavy">Intelligent Search:</h3><p>No need for a specific keyword and phrase.</p><br/>
              <h3 className="heavy">Relationship Analytics:</h3><p> Tagger learns about the relationship between you and your contacts to better prioritize your inbox search.</p><br/>
              <h3 className="heavy">Automated Tags:</h3><p>Machine Learning will automatically apply tags to better organize your emails and messages.</p><br/>
              <h3 className="heavy">Localized Environment:</h3><p> Your emails and credentials are stored locally on your computer for added privacy and security.</p><br/>
              <h3 className="heavy">Offline Capability:</h3><p> Access your email, compose new messages, and que up to be sent the moment you access the internet without prompting.</p><br/>
            </div>
          </div>
          <div className="about-image-container">
            <iframe
              className='media'
              src="https://drive.google.com/file/d/1hWgtgHQWiy11NycB7RYzcqBwC4UybmxO/preview"
              width="640"
              height="480"
            ></iframe>
          </div>
        </div>
      </div>
    </div>
  );
}
