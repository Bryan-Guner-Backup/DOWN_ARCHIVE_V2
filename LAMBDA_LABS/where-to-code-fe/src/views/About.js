import React, { useState, useEffect } from "react";
/* global google */

import Navigation from "../components/Navigation/index";

import styled from "styled-components";

import { withRouter} from "react-router-dom";

import mockup from '../assets/mockup.png';
import hours from '../assets/hours.jpg';
import explore from '../assets/explore.jpg';
import reviews from '../assets/reviews.jpg';

import * as ROUTES from "../Routes/routes";

const About = props => {

  return (
      <AboutPageContainer>
      <Navigation />
      <AboutScreen />
      <FeaturesInfo>
        <InfoBox>
          <h3>Work Easier</h3>
          <p>Finding a place to work away form the home or office doesn't need to be a hassle. Find locations that meet your needs no matter where you are!</p>
          <ExploreButton to={ROUTES.HOME}>Explore â†’</ExploreButton>
        </InfoBox>
        <FeatureBox>
          <FeaturePic src={explore} />
          <h3>Find New Places</h3>
          <p>Whether you're traveling or looking to get out of the house in your own home, we've got you covered.</p>
        </FeatureBox>
        <FeatureBox>
          <FeaturePic src={hours} />
          <h3>Get Info</h3>
          <p>Find the location's available hours, address, phone number, and more.</p>
        </FeatureBox>
        <FeatureBox>
          <FeaturePic src={reviews} />
          <h3>Pick the Best One</h3>
          <p>Read reviews from remote workers like you! Read about the speed and reliability of the internet, the noise level, and other important factors.</p>
        </FeatureBox>
      </FeaturesInfo>
      <MockupContainer>
          <Mockup src={mockup} />
        <LargeMarketing>
          <h3>Find Great Locations</h3>
          <p>Whether you're looking for some free wifi to work while you're traveling or you're just wanting to get away from the house or office for a change of scenery, The HiveStack is built to help you find the best place to work.</p>
          <Bullets>
            <div><i class="fa fa-coffee fa-2x"></i><p>Find coffee shops and cafes</p></div>
            <div><i class="fas fa-wifi fa-2x"></i><p>Check wifi speeds and reliability</p></div>
            <div><i class="fas fa-laptop fa-2x"></i><p>Sit down and get some work done</p></div>
          </Bullets>
        </LargeMarketing>
      </MockupContainer>
      </AboutPageContainer>
    
  );
};

export default withRouter(About);



const AboutPageContainer = styled.div``;

const AboutScreen = styled.div`
  background: url("/heroimage.svg");
  background-size: cover;
  display: flex;
  justify-content: center;
  align-items: center;
  width: 100%;
  height: 100vh;
`;

const FeaturesInfo = styled.div`
  margin: 125px 2% 20px 2%;
  display: flex;
  justify-content: space-between;
  @media(max-width: 500px){
    flex-direction: column;
  }
`

const InfoBox = styled.div`
  width: 23%;
  margin: 0 1%;
  h3 {
    font-size: 32px;
    font-family: "Zilla Slab", serif;
    font-weight: bold;
    margin-top: 0;
    padding-left: 10px;
  }
  p {
    font-size: 18px;
    color: #636363
    padding-left: 10px
  }
  @media(max-width: 500px){
    width: 95%;
    margin-bottom: 20px;
  }
`

const FeatureBox = styled(InfoBox)`
  border: 1px solid #ECECEC;
  h3 {
    font-size: 20px;
  }
  p {
    font-size: 14px;
  }
`
const FeaturePic = styled.img`
  width: 100%;
  height: 185px;
`

const MockupContainer = styled.div`
  display: flex;
  justify-content: space-around;
  @media(max-width: 500px){
    flex-direction: column;
  }
`

const Mockup = styled.img`
  width: 45%;
  margin: 50px 0;
  @media(max-width: 500px){
    width: 90%;
  }
`
const ExploreButton = styled(GoButton)`
  font-size: 1.1rem;
  border-radius: 10px;
  border-left: 1px solid gold;
  font-weight: bold;
  width: 125px;
  padding: 10px;
`

const LargeMarketing = styled(InfoBox)`
  width: 40%;
  margin-top: 80px;
`

const Bullets = styled.div`
  padding-left: 10px;
  display: flex;
  flex-direction: column;
  div {
    display: flex;
    align-content: center;
    margin-bottom: 10px;
    i {
      padding-right: 20px;
    }
    p{
      margin-top: 0;
      color: black;
      font-size: 16px;
    }
  }
`


