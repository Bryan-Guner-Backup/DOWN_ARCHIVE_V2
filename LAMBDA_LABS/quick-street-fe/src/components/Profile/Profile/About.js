import React, { useState } from "react"; //removed useEffect
import { VendorCategories } from "../../index";
import Map from "../../shared/Map";
//SVG Icons
import {
  pencil_light,
  phone_alt_light,
  envelope_light,
  map_marker_alt_light,
} from "../../../assets/svgs"; // removed pencil_regular, map_pin_place

// styling
import profile from "../../../styles/scss/vendor/a_vendors_profile.module.scss";
import about from "../../../styles/scss/vendor/a_vendors_about.module.scss";

const About = ({
  editAbout,
  vendorInfo,
  info,
  setInfo,
  editProfile,
  saveProfile,
  setVendorInfo,
}) => {
  const [editingNow, setEditingNow] = useState(null); //change back to none
  const [hoveringClass, setHoveringClass] = useState("not_hovering");

  const handleChanges = (e) => {
    setVendorInfo({
      ...vendorInfo,
      [e.target.name]: e.target.value,
    });
  };

  return (
    <div className={`${profile.wrapper} ${about.about_wrapper}`}>
      <div
        className={`${profile.inner_container} ${about.about_inner_container}`}
      >
        <div className={about.column_left}>
          {/* ABOUT US  */}
          <div className={about.about_info}>
            <div className={about.info_top}>
              <h2>About Us</h2>
            </div>
            {/* ==== TOP/BOTTOM divide ====== */}
            <div className={about.info_bottom}>
              <div className={about.info_content}>
                {editingNow === "about" ? ( // TURNARY ? show form...
                  <div>
                    <form action="">
                      <textarea
                        onChange={handleChanges}
                        name="description"
                        value={vendorInfo.description}
                        cols="30"
                        rows="10"
                      ></textarea>
                    </form>
                    <p
                      className={about.save_changes}
                      onClick={(e) => {
                        setEditingNow(null);
                        saveProfile(e);
                      }}
                    >
                      <i className="fa fa-save"></i>&nbsp;save
                    </p>
                  </div> //<<<<<<< TURNARY : else...show text
                ) : (
                  <div className={about.saved_text_container}>
                    {hoveringClass === "about_pen" && (
                      <img
                        className={`${about.edit_icon}`}
                        src={pencil_light}
                        alt=""
                      />
                    )}
                    <p
                      className={about.saved_text_content}
                      onMouseOver={() => setHoveringClass("about_pen")}
                      onMouseLeave={() => setHoveringClass("not_hovering")}
                      onClick={() => {
                        setEditingNow("about");
                        setHoveringClass("not_hovering");
                      }}
                    >
                      {vendorInfo.description}
                    </p>
                  </div>
                )}{" "}
                {/* END OF TURNARY - ABOUT US */}
              </div>
              <div className={about.edit_guides}>
                {editAbout && (
                  <p>
                    <i className="fa fa-save"></i>&nbsp; click to save
                  </p>
                )}
              </div>
            </div>
          </div>
          {/* END ABOUT US */}
          {/* HOURS OF OPERATION  */}
          <div className={about.about_info}>
            <div className={about.info_top}>
              <h2>Hours of Operation</h2>
            </div>
            {/* ==== TOP/BOTTOM divide ====== */}
            <div className={about.info_bottom}>
              <div className={about.info_content}>
                {editingNow === "hours" ? ( // TURNARY ? show form...
                  <div>
                    <form action="">
                      <textarea
                        onChange={handleChanges}
                        name="hours"
                        value={vendorInfo.hours}
                        cols="30"
                        rows="10"
                      ></textarea>
                    </form>
                    <p
                      className={about.save_changes}
                      onClick={() => {
                        setEditingNow(null);
                        saveProfile();
                        console.log("saveProfile() Profile.js info_content p ");
                      }}
                    >
                      <i className="fa fa-save"></i>&nbsp;save
                    </p>
                  </div> //<<<<<<< TURNARY : else...
                ) : (
                  <div className={about.saved_text_container}>
                    {hoveringClass === "hours_pen" && (
                      <img
                        className={`${about.edit_icon}`}
                        src={pencil_light}
                        alt=""
                      />
                    )}
                    <p
                      className={about.saved_text_content}
                      onMouseOver={() => setHoveringClass("hours_pen")}
                      onMouseLeave={() => setHoveringClass("not_hovering")}
                      onClick={() => {
                        setEditingNow("hours");
                        setHoveringClass("not_hovering");
                      }}
                    >
                      {vendorInfo.hours}
                    </p>
                  </div>
                )}{" "}
                {/* END OF TURNARY - HOURS OF OPERATION */}
              </div>
              <div className={about.edit_guides}>
                {editAbout && (
                  <p>
                    <i className="fa fa-save"></i>&nbsp; click to save
                  </p>
                )}
              </div>
            </div>
          </div>
          {/* END HOURS OF OPERATION */}
          {/* CONTACT */}
          <div className={about.about_info}>
            <div className={about.info_top}>
              <h2>Contact</h2>
            </div>
            {/* ==== TOP/BOTTOM divide ====== */}
            <div className={about.info_bottom}>
              <div className={about.info_content}>
                {editingNow === "phone" ? ( // TURNARY ? show form...
                  <div>
                    <form action="">
                      <input
                        onChange={handleChanges}
                        name="phone"
                        value={vendorInfo.phone}
                        cols="30"
                        rows="10"
                      ></input>
                    </form>
                    <p
                      className={about.save_changes}
                      onClick={(e) => {
                        setEditingNow(null);
                        saveProfile(e);
                      }}
                    >
                      <i className="fa fa-save"></i>&nbsp;save
                    </p>
                  </div> //<<<<<<< TURNARY : else...
                ) : (
                  <div className={about.saved_text_container}>
                    {hoveringClass === "phone_pen" && (
                      <img
                        className={`${about.edit_icon}`}
                        src={pencil_light}
                        alt=""
                      />
                    )}
                    <div className={about.contact_svg_p_container}>
                      <img
                        className={`${about.contact_icon}`}
                        src={phone_alt_light}
                        alt=""
                      />
                      <p
                        className={about.saved_text_content}
                        onMouseOver={() => setHoveringClass("phone_pen")}
                        onMouseLeave={() => setHoveringClass("not_hovering")}
                        onClick={() => {
                          setEditingNow("phone");
                          setHoveringClass("not_hovering");
                        }}
                      >
                        {vendorInfo.phone}
                      </p>
                    </div>
                  </div>
                )}{" "}
                {/* END OF TURNARY - PHONE*/}
                {editingNow === "email" ? ( // TURNARY ? show form...
                  <div>
                    <form action="">
                      <input
                        onChange={handleChanges}
                        name="email"
                        value={vendorInfo.email}
                        cols="30"
                        rows="10"
                      ></input>
                    </form>
                    <p
                      className={about.save_changes}
                      onClick={(e) => {
                        setEditingNow(null);
                        saveProfile(e);
                      }}
                    >
                      <i className="fa fa-save"></i>&nbsp;save
                    </p>
                  </div> //<<<<<<< TURNARY : else...
                ) : (
                  <div className={about.saved_text_container}>
                    {hoveringClass === "email_pen" && (
                      <img
                        className={`${about.edit_icon}`}
                        src={pencil_light}
                        alt=""
                      />
                    )}
                    <div className={about.contact_svg_p_container}>
                      <img
                        className={`${about.contact_icon}`}
                        src={envelope_light}
                        alt=""
                      />
                      <p
                        className={about.saved_text_content}
                        onMouseOver={() => setHoveringClass("email_pen")}
                        onMouseLeave={() => setHoveringClass("not_hovering")}
                        onClick={() => {
                          setEditingNow("email");
                          setHoveringClass("not_hovering");
                        }}
                      >
                        {vendorInfo.email}
                      </p>
                    </div>
                  </div>
                )}{" "}
                {/* END OF TURNARY - EMAIL*/}
              </div>
              <div className={about.edit_guides}>
                {editAbout && (
                  <p>
                    <i className="fa fa-save"></i>&nbsp; click to save
                  </p>
                )}
              </div>
            </div>
          </div>
          {/* END CONTACT */}
        </div>

        {/* ========   COLUMN LEFT/RIGHT DIVIDE ========= */}
        {/* ========   COLUMN LEFT/RIGHT DIVIDE ========= */}
        {/* ========   COLUMN LEFT/RIGHT DIVIDE ========= */}
        <div className={about.column_right}>
          {/*LOCATION AND MAP */}
          <div className={about.about_info}>
            <div className={about.info_top}>
              <h2>Location</h2>
              <img
                className={`${about.map_pin_icon}`}
                src={map_marker_alt_light}
                alt=""
              />
            </div>
            {/* ==== TOP/BOTTOM divide ====== */}
            <div className={about.info_bottom}>
              <div className={about.info_content}>
                {editingNow === "zipcode" ? ( // TURNARY ? show form...
                  <div>
                    <form action="">
                      <input
                        onChange={handleChanges}
                        name="zipcode"
                        value={vendorInfo.zipcode}
                        cols="30"
                        rows="10"
                      ></input>
                    </form>
                    <p
                      className={about.save_changes}
                      onClick={(e) => {
                        setEditingNow(null);
                        saveProfile(e);
                      }}
                    >
                      <i className="fa fa-save"></i>&nbsp;save
                    </p>
                  </div> //<<<<<<< TURNARY : else...show text
                ) : (
                  <div className={about.saved_text_container}>
                    {hoveringClass === "zipcode_pen" && (
                      <img
                        className={`${about.edit_icon}`}
                        src={pencil_light}
                        alt=""
                      />
                    )}
                    <p
                      className={about.saved_text_content}
                      onMouseOver={() => setHoveringClass("zipcode_pen")}
                      onMouseLeave={() => setHoveringClass("not_hovering")}
                      onClick={() => {
                        setEditingNow("zipcode");
                        setHoveringClass("not_hovering");
                      }}
                    >
                      {vendorInfo.zipcode}
                    </p>
                  </div>
                )}{" "}
                {/* END OF TURNARY - LOCATION */}
              </div>
              <div className={about.edit_guides}>
                {editAbout && (
                  <p>
                    <i className="fa fa-save"></i>&nbsp; click to save
                  </p>
                )}
              </div>
            </div>

            {/* <div className={about.map_container}>
              <Map zipcode={vendorInfo.location.zipcode} radius={3000} />
            </div> */}
          </div>
          {/* END LOCATION */}
        </div>
      </div>
    </div>
  );
};

export default About;
