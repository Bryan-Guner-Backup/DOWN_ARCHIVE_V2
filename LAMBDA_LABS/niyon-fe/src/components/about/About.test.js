import React from 'react'
import toJSON from 'enzyme-to-json'
import { shallow } from 'enzyme'
import About from './About'
import findByTestAttr from '../../tests/utils'

const setUp = (props = {}) => {
  const component = shallow(<About {...props}/>)
  return component
}

describe('<About /> component testing', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })

  it('should pass the snap shot test', () => {
    expect(toJSON(component)).toMatchSnapshot()
  })

  it('should render <About /> component correctly', () => {
    expect(component.length).toBe(1)
  })

  it('should render the <Navbar /> component correctly', () => {
    expect(component.find('Navbar').length).toBe(1)
  })
  it('should render  container div with no errors', () => {
    const container = findByTestAttr(component, 'aboutUs-container')
    expect(container.length).toBe(1)
  })

  it('should render title of the  component correctly', () => {
    expect(component.find('h1').text()).toEqual('Meet the Crew')
  })
  it('should render <PersonCard /> component correctly', () => {
    expect(component.find('PersonCard')).toHaveLength(1)
  })
})
