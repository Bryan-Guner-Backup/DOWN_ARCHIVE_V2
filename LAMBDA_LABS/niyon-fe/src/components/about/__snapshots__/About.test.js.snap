// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> component testing should pass the snap shot test 1`] = `
<div>
  <Navbar />
  <div
    className="container"
    data-test="aboutUs-container"
  >
    <h1>
      Meet the Crew
    </h1>
    <PersonCard />
  </div>
</div>
`;
